src/components/task-plan/homework/exercises.cjsx

```diff
@@ -33,6 +33,9 @@ ExerciseCardMixin =
   renderExercise: ->
     content = @props.exercise.content
     question = content.questions[0]
+    question.answers = _(question.answers).chain()
+      .sortBy('id')
+      .value()
     renderedAnswers = _.map(question.answers, @renderAnswers)
```

Comment:
Unfortunately, or fortunately, the answer ids are strings.  `.sortBy('id')` will be different from `.sortBy( (answer) ->
  parseInt(answer.id)
)`

in cases where there are answer ids '1234' and '123' for one question for example